tests/test/org/commcare/backend/session/test/SessionStackTests.java

```diff
@@ -20,24 +20,21 @@
 public class SessionStackTests {
     MockApp mApp;
 
-    @Before
-    public void init() throws Exception{
-        mApp = new MockApp("/complex_stack/");
-    }
-
     @Test
```

Comment:
All of the pre-existing tests in this file had the 'expected' and 'actual' arguments to assertEquals() swapped, so fixed that.